ktesting/testinglogger.go

```diff
@@ -61,14 +89,145 @@ func NewLogger(t TL, c *Config) logr.Logger {
 		prefix: "",
 		values: nil,
 		config: c,
+		buffer: new(buffer),
 	})
 }
 
+// Buffer stores log entries as formatted text and structured data.
+// It is safe to use this concurrently.
+//
+// Experimental
+//
+// Notice: This interface is EXPERIMENTAL and may be changed or removed in a
+// later release.
+type Buffer interface {
+	// String returns the log entries in the klog text
+	// output. The logger options determine whether
```

Comment:
```suggestion
	// String returns the log entries in the text format
	// The logger options determine whether
```

nit: Data returned by this is not really a klog text right? It is similar to that in most part but not exactly the same.

Reply:
Except for the header it is the same, so I guess it would depend on what is considered "the" klog text format.

But wording is slightly broken. How about "String returns the log entries in a format that is similar to the klog text output"?

Reply:
> string returns the log entries in a format that is similar to the klog text output

Absolutely. Sounds right to me.

Reply:
Also, printing the header is off unconditionally. Fixed that comment.

Reply:
This can be closed.